universe/config.json

```diff
@@ -69,12 +69,12 @@
 				"data_center_url": {
 					"id": "http://cassandra/docs/mesosphere.com/service/data_center_url",
 					"type": "string",
-					"description": "The URL of from which the the data center for the cluster managed by this service instance will server its data center information."
+					"description": "The URL from which the data center for the cluster managed by this service instance will server its data center information."
```

Comment:
Another good catch.  I have opened a separate ticket to comb through all of these messages...  I'm sure there are more.